packages/java/parser-jvm-core/src/main/java/dev/hilla/parser/models/TypeArgumentSourceModel.java

```diff
@@ -37,4 +50,17 @@ protected List<SignatureModel> prepareAssociatedTypes() {
         return signature == null ? List.of()
                 : List.of(SignatureModel.of(signature));
     }
+
+    private Stream<AnnotationInfoModel> getWildcardAnnotationsStream() {
+        // FIXME: workaround for
+        // https://github.com/classgraph/classgraph/issues/741,
+        // remove when the issue is fixed.
+        if (getWildcard().equals(TypeArgument.Wildcard.NONE)) {
+            return Stream.empty();
+        }
+        var strings = List.of(origin.toString().split(" "));
+        strings = strings.subList(0, strings.indexOf("?"));
+        return AnnotationInfoArtificialModel
```

Comment:
Just to be more consistent here
```suggestion
        return AnnotationInfoModel
```